ISHPullUp/ISHPullUpViewController.m

```diff
@@ -373,16 +378,77 @@ - (void)updateCachedHeightsWithSize:(CGSize)size  {
     self.maximumBottomHeightCached = [self maximumBottomHeightWithSize:size];
 }
 
+- (void)setBottomHidden:(BOOL)bottomHidden {
+    [self setBottomHidden:bottomHidden animated:NO];
+}
+
+- (void)setBottomHidden:(BOOL)bottomHidden animated:(BOOL)animated {
+    if (bottomHidden == _bottomHidden) {
+        return;
+    }
+
+    _bottomHidden = bottomHidden;
+    [self updateGestureEnabledState];
+
+    UIViewController *bottomVC = self.bottomViewController;
+
+    if (!bottomHidden) {
+        bottomVC.view.hidden = NO;
+        [bottomVC viewWillAppear:animated];
+    } else {
+        [bottomVC viewWillDisappear:animated];
+    }
+
+    void (^updateBlock)();
+    updateBlock = ^{
+        [self setDimmingViewHidden:[self dimmingViewShouldBeHidden] height:self.bottomHeight];
```

Comment:
The update block is executed immediately, I will keep as is as we usually skip this extra layer for animation blocks